core/src/main/resources/mapfish-spring-application-context.xml

```diff
@@ -64,12 +64,18 @@
 
     <bean id="forkJoinPool" class="jsr166y.ForkJoinPool" destroy-method="shutdownNow">
         <!--
-            Number of threads to user in pool we want a large number because http requests will block make the thread hang until
-            the request completes.  Because of this we want many threads available to the processors.
+            Thread pool used to run processors.
         -->
         <constructor-arg index="0" value="100" />
     </bean>
 
+    <bean id="requestForkJoinPool" class="jsr166y.ForkJoinPool" destroy-method="shutdownNow" autowire-candidate="false">
+        <!--
+            Thread pool used for making requests for tile/image layers.
+        -->
+        <constructor-arg index="0" value="15" />
```

Comment:
Please take 100 and 15 from properties defined in mapfish-spring.properties. That way it will be easier to tune.